orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/DisableServerGroupStage.groovy

```diff
@@ -33,13 +34,14 @@ import org.springframework.stereotype.Component
 @Component
 @CompileStatic
 class DisableServerGroupStage extends TargetServerGroupLinearStageSupport implements ForceCacheRefreshAware {
+  public static final String TOGGLE = "stages.disable-server-group.wait-for-disabled.enabled"
```

Comment:
you don't want/need `enabled` if you are using `dynamicConfig.isEnabled()` as it will add append it.
There was some talk of using Spring's `Environment` instead of `DynamicConfigService` I think it removes that ambiguity..?

```suggestion
  public static final String TOGGLE = "stages.disable-server-group.wait-for-disabled"
```

Reply:
I changed `dynamicConfig.isEnabled()` to not add it if it's already there because I strongly dislike that it adds it. I'm of the opinion that properties should be greppable

Reply:
You can always just use `Environment` instead. There's really no reason to use `DynamicConfigService`.

Reply:
alright then, `Environment` it is. Thanks!